### PR TITLE
i18n(es): Translated `from-gridsome.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `from-gridsome.mdx` to spanish

### CODE REVIEW

File: src/content/docs/es/guides/migrate-to-astro/from-gridsome.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+- [MPA vs SPA](/es/concepts/mpa-vs-spa/): Gridsome es una aplicación de una sola página (SPA) construida en Vue. Los sitios de Astro son aplicaciones de varias páginas construidas utilizando [componentes `.astro`](/es/core-concepts/astro-components/), pero también puede admitir [React, Preact, Vue.js, Svelte, SolidJS, AlpineJS, Lit](/es/core-concepts/framework-components/) y plantillas de HTML sin procesar.
+
+- Como una SPA, Gridsome utiliza `vue-router` para el enrutamiento SPA, y `vue-meta` para gestionar la etiqueta `<head>`. En Astro, crearás páginas HTML separadas y controlarás la etiqueta `<head>` de cada página directamente, o a través de un [componente de diseño](/es/core-concepts/layouts/).
+
+- [Datos de archivos locales](/es/guides/imports/): Gridsome utiliza GraphQL para recuperar los datos de tu proyecto. Astro usa importaciones ESM y el helper [`Astro.glob()`](/es/guides/imports/#astroglob) para importar los datos de archivos locales del proyecto. Los recursos remotos pueden cargarse utilizando el API estándar `fetch()`. GraphQL se puede añadir opcionalmente a su proyecto, pero no está incluido por defecto.
```
Comment (dreyfus92) [inline src/content/docs/es/guides/migrate-to-astro/from-gridsome.mdx:34]:
```suggestion
- [Datos de archivos locales](/es/guides/imports/): Gridsome utiliza GraphQL para recuperar los datos de tu proyecto. Astro usa importaciones ESM y el helper [`Astro.glob()`](/es/guides/imports/#astroglob) para importar los datos de archivos locales del proyecto. Los recursos remotos pueden cargarse utilizando el API estándar `fetch()`. GraphQL se puede añadir opcionalmente a tu proyecto, pero no está incluido por defecto.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59, te dejo algunas sugerencias.

**Comment by Waxer59** [comment]
Listo!, gracias por la revisión

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🙌

**Pull request merged**